### ISSUE TITLE
｢Examples｣ link is weird

### ISSUE THREAD

**Issue by AlexDaniel** [perl6/doc#710]
｢Language｣, ｢Types｣, ｢Routines｣ and ｢Programs｣ point to different sections on the website, but ｢Examples｣ gives a completely different website on a different domain.

Either move examples to doc.perl6.org or move that link somewhere else.

You may count it as part of https://github.com/perl6/doc/issues/665.

**Comment by zoffixznet** [comment]
If anyone is moving the link, for record, it's defined as a redirect in .htaccess:https://github.com/perl6/doc/blob/master/html/.htaccess#L12L13

**Comment by paultcochrane** [comment]
The examples site is a separate project from the documentation effort and hence should remain on examples.perl6.org.  Hence it's probably best to move the link somewhere else as suggested.

**Comment by AlexDaniel** [comment]
Homepage provides a link to design docs, so can we move the link to example there as well? Or would it make it not prominent enough?

**Comment by gfldex** [comment]
perl6.org is drowning in links. Ideally we would have a howto/idioms section in the docs where we either link to examples or have our own set of extended examples. That would be a good project _after_ we completed the rest of the docs. For now it's a nice friendly link that will hopefully provide some knowledge for newcomers. I learned Perl 5 with the Cookbook, so having examples that link to the appropriate sections in the docs would be really nice.
